enhancements/network/multus-services.md

```diff
@@ -0,0 +1,239 @@
+Kubernetes network solution satisfies the following condition, then this feature does not work:
+
+- The SDN solution provides forwarding plane for Kubernetes service (i.e. Cilium, ovn-kubernetes).
+- The SDN solution does not recognize the well-known label, `service.kubernetes.io/service-proxy-name`.
+
+Hence we should mention which SDN solution supports this enhancement in documents.
+
+## Design Details
+
+### Open Questions [optional]
+
+N/A
+
+### Test Plan
+
+For upstream testing, we should address e2e test in upstream in some fashion, however, upstream test will be done
+with `kind`, hence we could cover some of primitive testing. More detailed testing should be done in baremetal CI
+job or SR-IOV CI job with SR-IOV devices.
+
+### Graduation Criteria
+
+This feature, multus service abstraction, is fairly complicated because additional networks (i.e. network attachment
+definition) depend on customer environments, so each customer has each network and these are unlikely to be the same.
+So we require customer feedback not only about its quality also about their use-case and their experiences. We
+should watch customer feedback and carefully think about the current status of the feature and proceed to graduation.
+
+#### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users; rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+
+- Gain consensus regarding design and implementation among upstream communities
+- More testing (upgrade, downgrade, scale)
+- Sufficient feedback to cover various use-cases
+- Available by default
+- Conduct load testing
+- End to end tests in CI
+
+#### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+N/A
+
+### Version Skew Strategy
+
+N/A: This feature is not particularly kubernetes-version sensitive.
+
+## Implementation History
+
+This is dependent on the provision of gaining consensus, customer feedback as well as technical gaps which may be uncovered during.
+
+- *4.10*: Developer Preview
+- *4.11*: Tech Preview
+- *TBD*: GA
+
+## Drawbacks
+
+- Redesign users' infrastructure (including network) not to use multus (i.e. additional secondary networks)
+
+## Alternatives
+
+- Implement secondary network and service feature in ovn-kubernetes (but of course, it cannot support macvlan and other interface)
+
+### Required for GA
+
+GA requires following testing infrastructure:
+
+- Baremetal OpenShift deployment (for scale testing), with multiple network (macvlan/ipvlan/sr-iov)
+- Baremetal OpenShift deployment (for CI e2e testing), with multiple network. It might be possible to implement in equinix metal. Need to work on.
```

Comment:
Rework "Need to work on."

Reply:
It might be comment for the drafts. Removed.